activation/poet.go

```diff
@@ -127,8 +128,13 @@ func NewHTTPPoetClient(target string, cfg PoetConfig) *HTTPPoetClient {
 		return false, nil
 	}
 
+	// if no prefix given assuming http://
+	if !strings.HasPrefix(target, "http://") && !strings.HasPrefix(target, "https://") {
+		target = fmt.Sprintf("http://%s/v1", target)
+	}
```

Comment:
This block is confusing to me and I think it will be for users too.

IIUC for a poet server at `http://mydomain.com` as a user I could provide the following in the config
* `http://mydomain.com/v1`
* `mydomain.com`
but not
* `http://mydomain.com`

I'd rather not have 2 different valid strings for the address of a server, it makes it difficult to compare configs and tends to confuse users because they don't know what is the "right" way to config.

And the fact that `http://mydomain.com/v1` &  `mydomain.com` are accepted but `http://mydomain.com` will not be, I think would confuse users even more.

Reply:
Yes, I'll remove that completely. ANd then fix tests.